spec/features/outcome_page_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature "Display results of Rock, Paper Scissors" do
+  scenario "The computer chooses randomly and displays choice" do
+    register_name_and_play
```

Comment:
What is this and how do I use it? I just googled it and got 0 results!!!!! That has never happened to me. It looks like a much cleaner version of doing the "before" option that I used...